src/main/java/it/gov/pagopa/payhub/activities/exception/PaymentsClassificationSaveException.java

```diff
@@ -0,0 +1,19 @@
+package it.gov.pagopa.payhub.activities.exception;
+
+/**
+ * A custom exception that indicates the absence of an expected ingestion flow
+ * in the application's processing logic.
+ *
+ */
+public class PaymentsClassificationSaveException extends ActivitiesException {
```

Comment:
Probably we will not need such exception

Reply:
Yes, deleting exceptions in implementation of activity, i have also deteted this exception